src/supertux/gameconfig.hpp

```diff
@@ -79,8 +79,8 @@ class Config
       means autodetect. */
   std::string locale;
 
-  KeyboardConfig keyboard_config;
-  JoystickConfig joystick_config;
+  std::vector<std::shared_ptr<KeyboardConfig>> keyboard_configs;
```

Comment:
If these shared pointers are not used outside of this class, consider making them unqiue pointers.

Reply:
I made these pointers shared not because they are necessarily used outside of this class, but because of all of this class' methods and members are public, meaning it could be used outside of this class.

Reply:
Ok, lets keep them shared